src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -62,15 +62,29 @@ class MoreInfoMediaPlayer extends LitElement {
         ? ""
         : html`
             <div class="controls">
-              ${controls!.map(
-                (control) => html`
-                  <ha-icon-button
-                    action=${control.action}
-                    .icon=${control.icon}
-                    @click=${this._handleClick}
-                  ></ha-icon-button>
-                `
-              )}
+              <div class="basic-controls">
+                ${controls!.map(
+                  (control) => html`
+                    <ha-icon-button
+                      action=${control.action}
```

Comment:
We should add titles here too in a future PR

Reply:
And to the Media card as well